modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/webserver/test/WebServerFriendlyURLTest.java

```diff
@@ -69,6 +70,23 @@ public void testExistingFileEntryFriendlyURLHasFiles() throws Exception {
 		Assert.assertTrue(WebServerServlet.hasFiles(mockHttpServletRequest));
 	}
 
+	@Test
+	public void testExistingFileEntryFriendlyURLHasFilesParts2()
```

Comment:
Can you use proper names for the tests? e.g. `testGroupIdUUIDFriendlyURLHasFiles`.

Reply:
cheers for the suggestion @adolfopa , implemented